vermin/rules.py

```diff
@@ -3199,10 +3267,13 @@ def MOD_MEM_REQS():
 }
 
 # Builtin types that work as annotation types from 3.9+
+# The list is taken from Lib/test/test_genericalias.py in the CPython repository
```

Comment:
It would be nice to add the URL so that it's very easy to find: https://github.com/python/cpython/blob/master/Lib/test/test_genericalias.py